htdocs/openwebrx.js

```diff
@@ -783,6 +807,9 @@ function on_ws_recv(evt) {
                         if ('tuning_precision' in config)
                             $('#openwebrx-panel-receiver').demodulatorPanel().setTuningPrecision(config['tuning_precision']);
 
+                        if ('tuning_step' in config)
+                            tuning_step = config['tuning_step'];
+
```

Comment:
Will fix.

Reply:
Fixed in the last commit.

Reply:
Going to roll back the 'else' change, since this is not how on_ws_recv() function works. The config set it gets may contain any number of items. Items not passed via config set should not be modified in on_ws_recv() (see handling for other items from that set).

Reply:
You're right, I'm sorry. Mea culpa. I've even done this wrong myself...

Config changes are incremental, so indeed, the absence of a key does not mean that it should reset to the default.

I'll try to find the discussion about mandatory vs. optional keys first because this kind of hinges on that.